### DISCUSSION TITLE
Full screen button on a simple page

### DISCUSSION THREAD

**Discussion by ekiledda** [google/model-viewer#4338] [Q&A]
Hi all, I need a little bit of help with what probably is a simple thing. 
We're working on a virtual tour for a local museum; the idea is to add a few 3d models which we obtained with photogrammetry. The models work fine on a normal webpage, but when added to Panotour none of them is where we expect them to be (centre of the panotour window, ideally). How would i centre that? The html is basically just the script for the viewer and the aSide for the model + a css style page which should define were the model goes (which I've tried to modify but what I see on a webpage isn't what I see on the pop up window on panotour).
The colleague working on that thinks that we could at least add a full screen button, so you could see the model there but I've tried various script and I seem incapable to make any of them work. Any suggestion on how to make one that works? Bear in mind none of us is an expert on this, so ideally I need something easy to use. 

Any help/suggestion/alternatives highly appreciated, thank you all in advance

**Comment by WadeMorrison** [comment]
Could you possibly share some screenshots/videos/or even a demo of what you're working with if you haven't already figured out a solution?

**Comment by ekiledda** [comment]
Sure, i can share a couple of screenshoots. 
https://imgur.com/DxWQ88A 
https://imgur.com/zKazD3o

Those are the pop up windows we're currently using, and that's the model which isn't where we'd like it to be

**Reply by elalish** [reply]
This is a classic CSS problem because there are so many ways to center divs. Anything that works for a sized div should work for MV too. Here's some ideas: https://css-tricks.com/centering-css-complete-guide/

**Reply by ekiledda** [reply]
I was afraid that would be the answer. I did spend some time changing the css, but I couldn't move it like I wanted (I even got it out of the screen on the left and couldn't get it back in the middle). 
That link helped, probably. I think I need to create a parent which is going to be in relative position (so it can adapt to the window I have) while the model itself is fixed inside. Is that correct? 
 Currently my css is like this 
 #container {
    height: 100vh;
    width: 100%;
    position: relative;
    background: #f7f7f7;
}

#aSide {
    height: 600px;
    width: 600px;
    position: absolute;
    right: 60%;
}


#aSide model-viewer {
    height: 600px;
    width: 600px;
    position: absolute;
    top: 0;
    left: 0;
    border: none;
}

and I genuinely don't understand where I'm supposed to insert the parent/child thing.
